src/getPubkey.c

```diff
@@ -0,0 +1,80 @@
+#include "getPubkey.h"
+#include "os.h"
+#include "ux.h"
+#include "utils.h"
+
+static uint8_t publicKey[FULL_PUBKEY_LENGTH];
+
+static int read_bip32(const uint8_t *dataBuffer, size_t size, uint32_t *bip32) {
```

Comment:
This either needs a new name or better docs.  BIP32 is a standard. What's a `uinit32_t *bip32`?